NaReguaApp/src/routes/auth.routes.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Cadastro from '../paginas/cadastro';
+import Login from '../paginas/login';
+
+const AuthStack = createNativeStackNavigator();
+
+function AuthRoute() {
+  return (
+    <AuthStack.Navigator>
+      <AuthStack.Screen name="Login" component={Login} options={{ headerShown: false }}/>
```

Comment:
Codacy has a fix for the issue: [Replace `·name="Login"·component={Login}·options={{·headerShown:·false·}}` with `⏎········name="Login"⏎········component={Login}⏎········options={{headerShown:·false}}⏎······`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
      <AuthStack.Screen
```